src/model.py

```diff
@@ -69,7 +69,7 @@ def get_frames(self):
 
     def classify_frames(self):
         frames = self.get_frames()
-        results = [(self.score(Worker().classify_img(f)), t) for (f, t) in frames]
+        results = [(t, self.score(Worker().classify_img(f))) for (f, t) in frames]
         norm = 100
         results = [(val / norm, t) for (val, t) in results]
```

Comment:
Shouldn't the two be switched here (line 74), since the first element of each tuple in the returned list (in this commit, the timestamp) is normalized?

Reply:
Yes

Reply:
There was no issue because currently all scores are zero. See the fb chat.

Reply:
i.e. `results = [(t, val / norm) for (val, t) in results]` and undo the current changes.